src/main/java/frc/robot/commands/states/CleaningL2Reef.java

```diff
@@ -16,14 +18,15 @@ public class CleaningL2Reef extends Command {
   StateMachine globalStateMachine;
   Elevator globalElevator;
   AlgaeIntake globalAlgaeIntake;
+  LED subLED;
 
   /** Creates a new CleaningL2Reef. */
-  public CleaningL2Reef(StateMachine subStateMachine, Elevator subElevator, AlgaeIntake subAlgaeIntake) {
+  public CleaningL2Reef(StateMachine subStateMachine, Elevator subElevator, AlgaeIntake subAlgaeIntake, LED globalLED) {
     // Use addRequirements() here to declare subsystem dependencies.
     globalStateMachine = subStateMachine;
     globalElevator = subElevator;
     globalAlgaeIntake = subAlgaeIntake;
-
+    subLED = globalLED;
```

Comment:
This is swapped in order and it is swapped in all the commands
```suggestion
    globalLED = subLED;
```